From: Robert Schwebel <r.schwebel@pengutronix.de>
Subject: use modern libtool macro

aclocal.m4:123: AC_PROG_LIBTOOL is expanded from...

Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>

---
# 20091228 rsc: mailed to radu.voicilas@gmail.com

 configure.ac |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: inotify-tools-3.13/configure.ac
===================================================================
--- inotify-tools-3.13.orig/configure.ac
+++ inotify-tools-3.13/configure.ac
@@ -12,7 +12,7 @@ AC_PROG_MAKE_SET
 
 # Checks for programs.
 AC_PROG_CC
-AC_PROG_LIBTOOL
+LT_INIT
 
 AC_PATH_PROG(DOXYGEN, doxygen, NO_DOXYGEN)
 
